### PR TITLE
Update 'About this Release' with analytics features

### PR DESCRIPTION
**Pull request by ruks**
## Purpose
Update 'About this Release' with analytics features

### CODE REVIEW

File: en/docs/getting-started/about-this-release.md [hunk 1]
```diff
@@ -43,17 +43,25 @@ The **WSO2 API Manager 3.2.0** is the **latest** **WSO2 API Manager release*
...
-      
-      The Publisher dashboard is separated as API and Business dashboards. The Developer Portal dashboard is renamed to Application Analytics, and the Admin dashboard is renamed to Reports.
+      Significant changes have been done on the Publisher Analytics dashboard to better suit it to different audiences. A new dashboard named Business Analytics has been created by combining several new widgets with those which initially existed in Publisher Analytics Dashboard to provide an unrestricted view on the APIs for Managerial users. A user with an internal/analytics role can access the new Dashboard to see Statistics for all APIs irrespective of whether those are access-controlled or not. Further, a specific set of Widgets chosen from Publisher Analytics has been used to create an API Analytics Dashboard, which now shows a limited set of Statistics relevant to a Publisher. A Publisher can use this Dashboard to view statistics for both APIs created by them or visible to them.
```
Comment (shaniR) [inline en/docs/getting-started/about-this-release.md:50]: Widgets - widgets

Comment (shaniR) [reply en/docs/getting-started/about-this-release.md:50]: Dashboard - dashboard

File: en/docs/getting-started/about-this-release.md [hunk 2]
```diff
@@ -43,17 +43,25 @@ The **WSO2 API Manager 3.2.0** is the **latest** **WSO2 API Manager release*
...
+      The Developer Portal Dashboard has been renamed to Application Analytics and Admin Dashboard to Reports, to reflect their functionality better.
```
Comment (shaniR) [inline en/docs/getting-started/about-this-release.md:51]: Let's reword it is "The Developer portal dashboard has been renamed to **Application Analytics** and the Admin dashboard has been renamed to **Reports**  to reflect their functionality better"**

File: en/docs/getting-started/about-this-release.md [hunk 3]
```diff
@@ -43,17 +43,25 @@ The **WSO2 API Manager 3.2.0** is the **latest** **WSO2 API Manager release*
...
                                                                          
-- Support for exporting analytics data 
+- Exporting data in CSV, PDF formats 
     
-      It is possible to use Widgets that contain tabular data to export data as a CSV or PDF. Users can export the current view and also share, analyze, or compare the current view with different views. As a result, it is possible to export custom or selected data sets.
+      This new feature is introduced to widgets to export data as a CSV or PDF. It supports all the widgets with tabular data. Users can export the current view and can be share, analyze, or compare with different views. It helps to export custom or selected data sets.
```
Comment (shaniR) [inline en/docs/getting-started/about-this-release.md:55]: can be  - can

File: en/docs/getting-started/about-this-release.md [hunk 4]
```diff
@@ -43,17 +43,25 @@ The **WSO2 API Manager 3.2.0** is the **latest** **WSO2 API Manager release*
...
+      
+- New Alert Page in Monitoring Dashboards
+
+      API Availability and Alert listing capabilities implemented in Monitoring dashboards. Since this information is more related to analytics, moved to the monitoring dashboard from the API Manager admin portal. It provides an overview of the API availability and filtering based on the availability status.
```
Comment (shaniR) [inline en/docs/getting-started/about-this-release.md:59]: moved to the montoring - > it has been moved to the montoring

File: en/docs/getting-started/about-this-release.md [hunk 5]
```diff
@@ -43,17 +43,25 @@ The **WSO2 API Manager 3.2.0** is the **latest** **WSO2 API Manager release*
 
 - Monitoring dashboard
 
-      Identifying errors in the system and taking the necessary action by monitoring it is vital in API Management. The Dev-Ops and managers can use the interactive widgets support to identify and drill down information related to the API's and the application's traffic patterns, errors, latencies, etc., and isolate the issues.
+      Having an idea about the overall health status of APIs and Services is essential for the proper functioning of an API-driven business. Monitoring Dashboard helps to achieve this objective by showing contextual information on APIs such as Latency, Throughput, Errors, and Traffic Volume. Operational and Management users can use this dashboard to identify operational errors, narrow down the root cause, or to identify common traffic trends. The overview page provides a holistic view by aggregating different views on APIs, and by following the drill-downs user can get a detailed picture of different perspectives.
```
[the remark this answers is not in the document]
Comment (ruks) [reply en/docs/getting-started/about-this-release.md]: Fixed

File: en/docs/getting-started/about-this-release.md (hunk 1 above)
Comment (ruks) [reply en/docs/getting-started/about-this-release.md]: Fixed

File: en/docs/getting-started/about-this-release.md (hunk 3 above)
Comment (ruks) [reply en/docs/getting-started/about-this-release.md]: Fixed

File: en/docs/getting-started/about-this-release.md (hunk 4 above)
Comment (ruks) [reply en/docs/getting-started/about-this-release.md]: Fixed

**Pull request merged**